homeassistant/components/kostal_plenticore/sensor.py

```diff
@@ -834,32 +834,24 @@ async def async_setup_entry(
         plenticore,
     )
     for description in SENSOR_PROCESS_DATA:
-        module_id = description.module_id
-        data_id = description.key
-        name = description.name
-        sensor_data = description.properties
-        fmt = description.formatter
         if (
-            module_id not in available_process_data
-            or data_id not in available_process_data[module_id]
+            description.module_id not in available_process_data
```

Comment:
I'd keep the local variables for module_id and data_id as we're using them 2 - 3 times here.

Reply:
done